app/code/Magento/InventoryInStorePickup/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd suggest putting mftf test related content into `*AdminUi` directories.

Reply:
Done. Test moved to InventoryInStorePickupAdminUi